Sources/Starknet/Accounts/StarknetAccount.swift

```diff
@@ -12,49 +16,52 @@ public class StarknetAccount: StarknetAccountProtocol {
         self.provider = provider
     }
 
-    private func makeSequencerInvokeTransaction(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams) -> StarknetSequencerInvokeTransaction {
-        StarknetSequencerInvokeTransaction(senderAddress: address, calldata: calldata, signature: signature, maxFee: params.maxFee, nonce: params.nonce)
+    private func makeSequencerInvokeTransaction(calldata: StarknetCalldata, signature: StarknetSignature, params: StarknetExecutionParams, version: Felt) -> StarknetSequencerInvokeTransaction {
+        StarknetSequencerInvokeTransaction(senderAddress: address, calldata: calldata, signature: signature, maxFee: params.maxFee, nonce: params.nonce, version: version)
     }
 
-    private func makeSequencerDeployAccountTransaction(classHash: Felt, salt: Felt, calldata: StarknetCalldata, maxFee: Felt, signature: StarknetSignature) -> StarknetSequencerDeployAccountTransaction {
+    private func makeSequencerDeployAccountTransaction(classHash: Felt, salt: Felt, calldata: StarknetCalldata, maxFee: Felt, signature: StarknetSignature, version: Felt) -> StarknetSequencerDeployAccountTransaction {
         StarknetSequencerDeployAccountTransaction(
             signature: signature,
             maxFee: maxFee,
             nonce: .zero,
             contractAddressSalt: salt,
             constructorCalldata: calldata,
-            classHash: classHash
+            classHash: classHash,
+            version: version
         )
     }
 
-    public func sign(calls: [StarknetCall], params: StarknetExecutionParams) throws -> StarknetSequencerInvokeTransaction {
+    public func sign(calls: [StarknetCall], params: StarknetExecutionParams, forFeeEstimation: Bool) throws -> StarknetSequencerInvokeTransaction {
```

Comment:
I think `forFeeEstimation` could be an optional parameter (the same in signDeployAccount)